client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -158,6 +221,13 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
                 .
             </Text>
             <Container className="mb-3">
+                {error && !loading && (
+                    <Alert variant="warning" as="p">
+                        {error.message}
+                    </Alert>
+                )}
+                {loading && !error && <LoadingSpinner />}
```

Comment:
does this work with polling? Wouldn't it flash a loader each time it polls?
Maybe this helps, if so:
```suggestion
                {loading && !error && !legends && <LoadingSpinner />}
```

Reply:
It doesn't. See demo video. But now I'll double-check.

Reply:
Nope. It works. Doesn't flash. Shows loading spinner on initial load.